packages/holocron/src/index.js

```diff
@@ -69,4 +74,8 @@ export {
   setRequiredExternalsRegistry,
   clearModulesUsingExternals,
   getModulesUsingExternals,
+  // Streaming
+  ReactStreamingContext,
+  ModuleContext,
```

Comment:
should these be public ? holocron hoc could include them

Reply:
I've moved ModuleContext to the holocron hoc, but the one-app server will still need to reference these.

Reply:
Can the one-app server reference them from holocron/server?

Reply:
It also need to be available in the client to ensure hydration works